backend/src/packages/chaiNNer_standard/image_utility/compositing/blend_images.py

```diff
@@ -23,18 +77,116 @@
         ImageInput("Base Layer", channels=[1, 3, 4], allow_colors=True),
         ImageInput("Overlay Layer", channels=[1, 3, 4], allow_colors=True),
         BlendModeDropdown(),
+        if_group(Condition.type(0, "Image") & Condition.type(1, "Image"))(
+            EnumInput(
+                BlendOverlayPosition,
+                label="Overlay position",
+                option_labels=BLEND_OVERLAY_POSITION_LABELS,
+                default_value=BlendOverlayPosition.CENTERED,
+            ),
+            if_enum_group(3, (BlendOverlayPosition.PERCENT_OFFSET))(
+                SliderInput(
+                    "X offset",
+                    precision=0,
+                    controls_step=1,
+                    minimum=-200,
+                    maximum=200,
+                    default=0,
+                    unit="%",
+                ),
+                SliderInput(
+                    "Y offset",
+                    precision=0,
+                    controls_step=1,
+                    minimum=-200,
+                    maximum=200,
+                    default=0,
+                    unit="%",
+                ),
+            ),
+            if_enum_group(3, (BlendOverlayPosition.PIXEL_OFFSET))(
+                NumberInput(
+                    "X offset",
+                    controls_step=1,
+                    minimum=None,
+                    maximum=None,
+                    default=0,
+                    unit="px",
+                ),
+                NumberInput(
+                    "Y offset",
+                    controls_step=1,
+                    minimum=None,
+                    maximum=None,
+                    default=0,
+                    unit="px",
+                ),
+            ),
+        ),
+        BoolInput("Crop to fit base layer", default=False),
```

Comment:
Shouldn't this one be inside the `if_group` as well? What does "Crop to fit base layer" even mean when one of the layers is a color?

Reply:
Right, mistake when I moved the checkbox at the bottom of the node.
No meaning when an input is a color